python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -292,14 +288,21 @@ def __deepcopy__(self, memo):
             new_list._is_preparable = self._is_preparable
             return new_list
 
-    @staticmethod
-    def __getListToCompare(input_list):
+    def __getListToCompare(self, input_list):
+
+        # if the arg isn't a list, just give it back
+        if not self.is_list(self.strip_proxy(input_list)):
+            return input_list
+
+        # setup up the list correctly
+        tmp_list = input_list
         if isType(input_list, GangaList):
-            return stripProxy(input_list)._list
```

Comment:
This is more correct as ```_list``` should never contain proxied objects. Changing this here is at least wasteful and potentially hiding problems elsewhere which need to be addressed.